Open-Source ASAM Structure (#4)
asam-ev/.github: profile/CONTRIBUTING.md

All intellectual property remains with the original contributors, and is subject to the original license terms described in the LICENSE file.

> ## How is it structured in ASAM?

Reviewer: How are open-source projects structured at ASAM?

Author: updated

ASAM recognizes several hosting modes:

> - The product is developed within an ASAM project group and released through the ASAM marketplace.

Reviewer: ```suggestion
- The product is developed within an ASAM project group and released under the ASAM GitHub organization.
```

>     The declaration document specifies that:

Reviewer: Which declaration document?

Reviewer: The below points are nice statements, but if it is created within an ASAM project that what is more important is Who guarantees them? In a tool created in an ASAM project, it is ASAM.

        1. The software is a result of the project group’s activities.
        2. No project group member retains any rights.
        3. All required components are provided.
        4. The deliverable is free from malware.
        5. The deliverable is relevant for ASAM standardization work.

> - The product is handed over to an ASAM project group for continued development(modification/extension) and jointly released via the ASAM marketplace.

Reviewer: There is no "joint release" here. This is the same release mode as above. The inputs are different but everything else still applies. I would combine the two approaches.

  
     The declaration document specifies that:

        1. The software is a result of the project group’s activities.
        2. No project group member retains any rights.
        3. For third-party components (select applicable items):

>             3.a The deliverable is free from third-party rights.

Reviewer: Is this meant to be "No third party components are included / N/A" ?

Author: With "free from 3rd-party rights" I think Marius was meaning that the product does not infringe on any intellectual property rights held by third parties (e.g., patents, copyrights, trademarks). So to make it sure that there are no legal claims or ownership issues from third parties regarding the product.

>             3.b All third-party components are open-source and comply with ASAM requirements.

Reviewer: which requirements?

Author: You are right, the requirements are actually what is written there. I will delete the end of the sentence

            3.c All third-party rights have been transferred to ASAM.
        7. All required components are provided.
        8. The deliverable is free from malware.
        9. The deliverable is relevant for ASAM standardization work.
  
- The product is transferred to ASAM for direct publication, with no modifications, as an ASAM product on the ASAM marketplace.
  

>     The transfer agreement specifies that:

Reviewer: Here you reference a transfer agreement, previously you referred a declaration document. Are these the same? This should be defined in these guidelines. This is an agreement between ASAM and the submitting party or parties to transfer content to ASAM, which outlines the points described in the bullets.

Reviewer: Paarticular emphasis on the rights and licensing. A license shall be defined in the the transfer document, under which all transferred material shall be licensed.

      - ASAM is authorized to:
  
            1. Provide public access
            2. Distribute free copies
            3. Modify and/or integrate with other software
            4. Reject or delete the deliverable
            5. Sub-license the software

>             6. Promote or advertise the deliverable

Reviewer: Redundant with the license selection

  
      - The third party guarantees:
  

>             1. Compliance with ASAM’s licensing terms

Reviewer: ```suggestion

```

            2. For third-party components (select applicable items):
                2.a The deliverable is free of third-party rights
                2.b All third-party components are open-source and comply with ASAM requirements
                2.c All third-party rights have been transferred to ASAM
            3. All necessary components are included
            4. The deliverable is free of malware
            5. The deliverable is relevant to ASAM standardization efforts
  
- The product is submitted directly to ASAM for publication as a third-party product, without any changes, on the ASAM marketplace.
  
    The marketplace terms and conditions (T&C) specify the following:
  
        - ASAM provides:
  
            1. A hosting platform (either directly or through a service provider)
  
        - The third party guarantees:
  
            1. The assets are provided at the third party's own risk
            2. The assets are free from malware
            3. The assets are free of any third-party rights
            4. The assets are fully available for free, with no hidden costs

>             5. The assets are relevant to ASAM standardization efforts.

Reviewer: This is repetitive and should be linked separately. A clear set of Marketplace T&Cs should be linked to in the organization profile. This way you can always state, independently of which hosting method is selected, that products on the marketplace are compliant with the marketplace T&Cs.

Author: I´m open to create another "page" just for the marketplace T&Cs. So in this hosting mode we can just write: "see marketplace T&Cs"

![tp header](/doc/img/OpenSource_ASAM_Overview_Platform.png)

The ASAM office will consistently conduct a compliance check to ensure all requirements are met.

> Finally, the marketplace terms and conditions also state that:

Reviewer: See prior comment on Marketplace T&Cs.

Author: We can put this disclaimer in the Marketplace T&Cs.

    - ASAM provides assets:
    
        1. Free of charge
        2. Under ASAM license conditions
        3. "As is," without any warranties
        4. Without a guarantee of suitability for the intended purpose
        
    - The end user agrees to:
    
        1. Comply with ASAM license conditions
        2. Cooperate in addressing any defects

> 

Reviewer: Mostly redundant as it depends on the license. What we should state are points 3 and 4 however. The typical "no liability" statement. For this, the common phrasing is in the suggestion below.

Author: agreed

>     - ASAM provides assets:
>     
>         1. Free of charge
>         2. Under ASAM license conditions
>         3. "As is," without any warranties
>         4. Without a guarantee of suitability for the intended purpose
>         
>     - The end user agrees to:
>     
>         1. Comply with ASAM license conditions
>         2. Cooperate in addressing any defects
> 

Reviewer: ```suggestion

"THE TOOLS AND CONTENT AVAILABLE ON THIS PLATFORM ARE PROVIDED 'AS IS' AND 'AS AVAILABLE', WITHOUT WARRANTIES OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO IMPLIED WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, NON-INFRINGEMENT, OR ACCURACY. THE PLATFORM MAKES NO GUARANTEES REGARDING THE COMPLETENESS, RELIABILITY, OR SUITABILITY OF ANY TOOL OR SERVICE FOR A PARTICULAR PURPOSE. ALL USE IS AT YOUR OWN RISK. THE PLATFORM AND ITS OPERATORS SHALL NOT BE LIABLE FOR ANY DAMAGES, LOSSES, OR CLAIMS ARISING FROM THE USE OF ANY TOOL OR CONTENT PROVIDED BY THIRD PARTIES ON THIS PLATFORM."
```

## How to officially [join ASAM](https://www.asam.net/about-asam/join-asam/) or the project

> Before you fill out the [Membership Request Form](https://www.asam.net/index.php?eID=dumpFile&t=f&f=671&token=b37651b1609e8693188bbde1dafc4f229c18c2c8) we ask you to get familiar with the [ASAM Statutes](https://www.asam.net/index.php?eID=dumpFile&t=f&f=675&token=5a112bf0f7638cbfd647f89b89869af94e212762) and the [Regulation of Fees](https://www.asam.net/index.php?eID=dumpFile&t=f&f=673&token=1321eb0a4e6cfb99112f61648e411520f83a8d01). If you have questions when determining your member class, please check the FAQs below or contact us at info@asam.net.

Reviewer: Overall I am missing text on what I actually need to do to get something submitted to ASAM? Where do I start?

Reviewer: I cannot review it, but line 106 is incorrect, the MPL license is not default. We should state something akin to the comment previously; a license is selected and defined during the transfer process.

Author: > I cannot review it, but line 106 is incorrect, the MPL license is not default. We should state something akin to the comment previously; a license is selected and defined during the transfer process.

we need to talk maybe a little bit more in detail about the ASAM License policies. Right now we say that we only accept projects that are licensed under MPL 2.0, Apache or MIT. are they all?

Author: > I cannot review it, but line 106 is incorrect, the MPL license is not default. We should state something akin to the comment previously; a license is selected and defined during the transfer process.

I´ve updated the text as following: "Projects transferred into our marketplace will undergo a defined transfer process, during which the final license will be selected and explicitly established"
